Userland/Libraries/LibWeb/UIEvents/WheelEvent.idl

```diff
@@ -0,0 +1,26 @@
+#import <UIEvents/MouseEvent.idl>
+
+// https://www.w3.org/TR/uievents/#idl-wheelevent
+[Exposed=Window]
+interface WheelEvent : MouseEvent {
+
```

Comment:
No need for these empty lines, please try to copy IDL from the spec verbatim and only fix up the indentation (I know some existing files don't adhere to this)

Reply:
yeah, maybe I followed pattern of other .idl files too precisely :) fixed